docs/_posts/2016-05-23-standards-are-the-future.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Standards are the Future"
+author: jimfb
+---
+
+Historically, popular frameworks had to build their own class systems, because Javascript didn’t have a built-in class system.  Starting in React v0.13, we announced that we would be switching to standard Javascript constructs.  We began supporting javascript classes before the standard `class` specification was even officially finalized.  In June of last year, the Javascript standards committee approved ES2015, which officially added classes to the Javascript language specification.  If you haven’t already started transitioning away from `React.createClass`, now is the time!  We are officially classifying `React.createClass` as legacy, and recommending that people move toward standard constructs like functions and classes.
+
+## ES6 Classes
+ES6 Classes, also known as “ES2015 Classes” or “Javascript Classes”, are the newly-introduced javascript standard for representing classes.  React components can be defined using standard classes, instead of using `React.createClass`.
+
+A React component defined using the class syntax looks like this:
+
+```js
+class MyComponent extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+  render() {
+    return <div>Hello {this.props.name}</div>;
+  }
+  componentDidMount() {
+    // we can override a lifecycle method.
```

Comment:
I’d change this to `// We can define a lifecycle method`
“Override” gives a wrong connotation that it’s defined in the base class and you must call `super`, which is not true.